hbase-server/src/main/java/org/apache/hadoop/hbase/replication/HBaseReplicationEndpoint.java

```diff
@@ -118,6 +118,7 @@
   private boolean fetchServersUseZk = false;
   private FetchServersChore fetchServersChore;
   private int shortOperationTimeout;
+  private boolean isReplicationServer = false;
```

Comment:
Yeah, boolean should be named replicationServer and the method to access this data member is named isReplicationServer.